app/assets/stylesheets/static_pages.scss

```diff
@@ -292,6 +292,317 @@ nav {
   background-size: contain;
 }
 
+#user-badge1  {
+    background: image-url('UserLogos/linux.png') no-repeat top left;
+  display:block;
+  width: 100px;
+  height: 100px;
+  background-size: contain;
+  text-indent:-9999px;
+}
```

Comment:
Putting the images into the CSS, instead of as img src, is *weird*.  But if you're going to do this, then let's make the names actually useful, that is, "linux-badge" not "user-badge1".

Reply:
Big problem: It doesn't work for me.  I don't see any logos.

Do you really need to put the image names in CSS?  It's more complex, and this is a *lot* of extra stuff.  Shouldn't there be a single class that sets width, height, and contain at least, and then just have everyone refer to that one class?